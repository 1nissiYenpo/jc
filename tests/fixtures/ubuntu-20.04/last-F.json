[{"user": "kbrazil", "tty": "pts/0", "hostname": "192.168.71.1", "login": "Tue Jan 5 20:03:51 2021", "logout": "still logged in", "login_epoch": 1609905831}, {"user": "kbrazil", "tty": "pts/0", "hostname": "192.168.71.1", "login": "Tue Jan 5 00:17:29 2021", "logout": "Tue Jan 5 20:03:47 2021", "duration": "19:46", "login_epoch": 1609834649, "logout_epoch": 1609905827, "duration_seconds": 71178}, {"user": "kbrazil", "tty": "tty1", "hostname": null, "login": "Tue Jan 5 00:08:46 2021", "logout": "gone - no logout", "login_epoch": 1609834126}, {"user": "root", "tty": "pts/0", "hostname": "developer-pc.exa", "login": "Mon Mar 23 16:44:55 2020", "logout": "down", "duration": "00:00", "login_epoch": 1585007095}, {"user": "reboot", "tty": "system boot", "hostname": "5.8.0-34-generic", "login": "Tue Jan 5 00:08:28 2021", "logout": "running", "login_epoch": 1609834108}]
