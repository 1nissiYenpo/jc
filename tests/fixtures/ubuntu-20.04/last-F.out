kbrazil  pts/0        192.168.71.1     Tue Jan  5 20:03:51 2021   still logged in
kbrazil  pts/0        192.168.71.1     Tue Jan  5 00:17:29 2021 - Tue Jan  5 20:03:47 2021  (19:46)
kbrazil  tty1                          Tue Jan  5 00:08:46 2021   gone - no logout
reboot   system boot  5.8.0-34-generic Tue Jan  5 00:08:28 2021   still running

wtmp begins Tue Jan  5 00:08:28 2021
